misc/stock_emails.md

```diff
@@ -0,0 +1,100 @@
+# Inquiries about joining Mozilla L10n
+
+Here you’ll find a stockpile of templated responses to inquiries we frequently receive about Mozilla L10n. They’ve been centralized here so that you can simply copy and paste them into emails. While there is value to having these stock responses for common inquiries, we strongly feel that there is equal value to making them as personalized as possible. Take time to adjust the template in your email to include personalized information.
+
+## General inquiry
+
+Hi [inquiree],
+
+Thank you for your email and welcome to the Mozilla Community! We’re excited that you’re interested in joining our localization (L10n) effort.
+
+Joining our L10n effort is easy. Within five easy steps you can start contributing to the project! We have a wiki page specially designed to help you get started. Please visit https://wiki.mozilla.org/L10n:Contribute and follow the five steps to get involved.
+
+Could you also please provide me with some additional information about yourself, such as languages you speak, country you live in, and technical background?
+
+You can also visit the following wiki pages to learn the details of how we localize:
+* [https://wiki.mozilla.org/L10n:Localization_Process L10n:Localization Process:] A good description of the whole process.
```

Comment:
The link is broken here. What's the URL you want to point to? Since they're email templates, I would just put the plain links all over the document, or the HTML version `<a href="URL">link text</a>".